src/components/SpruceForm/customFormats.ts

```diff
@@ -1,11 +1,26 @@
 import { validators } from "utils";
 
-const { validateURL, validateJiraURL } = validators;
+const {
```

Comment:
<img width="612" alt="image" src="https://user-images.githubusercontent.com/13104717/175552001-00374281-6559-47e5-b94a-d96397f227f6.png">
this shows an error despite being selected

Reply:
same here:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/13104717/175553998-d90ad943-c3ae-4c7a-89cd-ea1b6ae5103c.png">